payments/invoice/direct.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Swedbank Pay Payments Invoice Direct
+sidebar:
+  navigation:
+  - title: Invoice Payments
+    items:
+    - url: /payments/invoice
+      title: Introduction
+    - url: /payments/invoice/direct
+      title: Direct
+    - url: /payments/invoice/redirect
+      title: Redirect
+    - url: /payments/invoice/seamless-view
+      title: Seamless View
+    - url: /payments/invoice/after-payment
+      title: After Payment
+    - url: /payments/invoice/other-features
+      title: Other Features
+---
+
+{% include jumbotron.html body="**Direct**
+Direct is a payment service where Swedbank Pay helps improve cashflow by
+purchasing merchant invoices. Swedbank Pay receives invoice data, which is used
+to produce and distribute invoices to the consumer/end-use" %}
+
+## Introduction
+
+1. Collect all purchase information and send it in a `POST` request to
+Swedbank Pay.
+1. Include personal information (SSN and postal code) and send it to Swedbank Pay.
+1. Make a new `POST` request towards Swedbank Pay to retrieve the name and
+address of the customer.
+1. Create an authorization transaction by calculating the final price / amount.
+1. Make a third `POST` request with consumer data as input.
+1. Send a  `GET` request with the `paymentID` to get the authorization result
+1. Make a Capture by creating a `POST` request
+
+  **By making a Capture, Swedbank Pay will generate
+  the invoice to the consumer and the order is ready for shipping.**
+
+## Options before posting a payment
+
+All valid options when posting a payment with operation equal to
+`FinancingConsumer`, are described in
+[other features][other-features-financing-consumer].
+
+{:.table .table-striped}
+| | **Sweden** ![Swedish flag][se-png]| **Norway** ![Norwegian flag][no-png] | **Finland** ![Finish flag][fi-png] |
```

Comment:
Turn this into a proper table with a header/body separator line and then format the table. This applies to all tables.

```suggestion
| | Sweden ![Swedish flag][se-png]| Norway ![Norwegian flag][no-png] | Finland ![Finish flag][fi-png] |
| :------ | :------------------- | :------- | :------------------- |
```